### ISSUE TITLE
Refactor /recs builds filter to pull from builds mongo collection instead of recordedgames collection

### ISSUE THREAD

**Issue by erin-fitzpatric** [erin-fitzpatric/next-aom-stats#85]
The builds filter currently builds the list of builds by querying the recordedgames collection for unique buildnumbers. I already created a new build collection that populates a new record anytime a new build is detected in an upload.

1. Migration script to get all previous builds and add them to the build collection with the appropriate metadata
2. There is metadata that will need to be manually added (releaseDate)...I'm not even sure if we can find that. Maybe just build this to support new builds going forward? At the very least, we know the release dates of the path from last friday and the Red Bull build.
3. Update the build schema to include:
```
{
    description: { type: String, required: false }
}
```
4. The build number represents the athens internal build number, but isn't different from the steam branch, which is confusing for users. For now, builds will be updated with a manual description. This could be "Red Bull Qualifier Branch" or "Update 2.0"...whatever. 
5. I would like to use these descriptions in the front end, something like `1.4.5 Update - 9/16/24` or `1.2.3 RbW Branch - 9/17/24`, etc.

TLDR:
- update build mongo schema
- write/run a migration that will populate builds from previous uploads to the recordedgames collection
- manually enter descriptions, dates etc for the most recent builds
- update front end drop down to show friendly descriptions that the user will understand. {patchNumber - description - publishDate}
